app/controllers/collection_items_controller.rb

```diff
@@ -1,9 +1,7 @@
 class CollectionItemsController < ApplicationController
   before_action :load_collection
   before_action :load_user, only: [:update_multiple]
-  before_action :load_item_and_collection, only: [:destroy]
```

Comment:
Like `allowed_to_destroy`, you should be able to delete `load_item_and_collection` now that it's unused. (And as a consequence, you should be able to delete `collection_item_params` as well.)